DayrSuryan/DSEthiop22.xml

```diff
@@ -80,14 +80,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                         <foliation>The manuscript is not foliated.</foliation>
                         <collation>
-                           <note>The text block is a multi-quire large format notebook
```

Comment:
As I understood the result of https://github.com/BetaMasaheft/Documentation/issues/1418, we agreed to put this in `<p>` in `<support>`. Can I move it there?

Reply:
OK, of course.